src/Components/Blueprints/DeleteBlueprintModal.tsx

```diff
@@ -69,8 +69,8 @@ export const DeleteBlueprintModal: React.FunctionComponent<
       titleIconVariant="warning"
       isOpen={isOpen}
       onClose={onDeleteClose}
-      title={`Permanently delete ${blueprintName}?`}
-      description={'All versions will be lost.'}
+      title={'Delete blueprint?'}
+      description={`All versions of ${blueprintName} and its associated images will be deleted.`}
```

Comment:
will blueprint's associated images  be deleted? these are expired after 14 weeks anyway
AFAIK there is no delete cascading on associated images.

Reply:
@croissanne what do you think?